Makefile

```diff
@@ -8,8 +8,8 @@ deps:
 	go get github.com/tools/godep
 
 build: clean deps
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 godep go build -a ./... 
```

Comment:
This is useful when compiling on OSX. If we switch to docker based builds, we can get rid of this safely.